packages/account/src/test-utils/launchNodeAndGetWallets.test.ts

```diff
@@ -57,7 +57,7 @@ describe('launchNode', () => {
     const { stop, wallets } = await launchNodeAndGetWallets({
       walletCount: 5,
       providerOptions: {
-        cacheUtxo: 1,
+        resourceCacheTTL: 1,
       },
```

Comment:
```suggestion
```

It seems to me that there's no point in specifying this in the test.

Reply:
Ohh, this is the obsolete `launchNodeAndGetWallets` utility. Ignore the comment.